Source/Plugins/Core/com.equella.core/js/tsrc/search/components/SearchBar.tsx

```diff
@@ -16,26 +16,38 @@
  * limitations under the License.
  */
 import {
-  Card,
-  CardContent,
+  Divider,
   FormControlLabel,
-  Grid,
   IconButton,
-  InputAdornment,
+  InputBase,
+  Paper,
   Switch,
-  TextField,
   Tooltip,
 } from "@material-ui/core";
 import * as React from "react";
 import { useCallback, useState } from "react";
 import SearchIcon from "@material-ui/icons/Search";
-import { Close } from "@material-ui/icons";
 import { debounce } from "lodash";
 import { languageStrings } from "../../util/langstrings";
+import { makeStyles } from "@material-ui/core/styles";
 
 const ENTER_KEY_CODE = 13;
 const ESCAPE_KEY_CODE = 27;
 
+const useStyles = makeStyles({
+  root: {
+    display: "flex",
+    alignItems: "center",
+  },
+  input: {
+    flex: "auto",
+  },
+  divider: {
+    height: 28,
+    margin: "4px 12px 4px",
```

Comment:
Since top and bottom margin are the same this can be simplified to:
```suggestion
    margin: "4px 12px",
```